### ISSUE TITLE
Idea: Allow channels to be muted on both chips

### ISSUE THREAD

**Issue by renedare** [AidanHockey5/MegaBlaster2#12]
The MB2 is great, but it lacks the option to mute channels on the YM/SN chips.
This is an interesting feature to "deconstruct" tracks and listen to each instrument if we want to.

I know this is trivial in terms of needs to be done to the chip writes, but i'm not so sure about the UI.

**Comment by renedare** [comment]
I actually started experimenting with the source code and was able to create a sub-menu with this option. I'm more than happy to submit a pull request once I finish some extra testing. :)

**Comment by AidanHockey5** [comment]
Hello!

That sounds like a good feature and I’d more than happy to help research
it/implement your changes. I’m actually wrapping up a vacation on the other
side of the world right now, so I’ll be pretty silent these next couple
days. Once I get back and all settled in I’ll take a look. Feel free to
submit a pull request for review.

**Comment by renedare** [comment]
Cool! 

I've recorded a quick video with the preliminary implementation. I just noticed I had a bug in the channel decoding logic of the YM2612, the channels are numbered 0,1,2 then 4,5,6 (it skips the number 3).

https://www.youtube.com/watch?v=LfFcEJUGSFE

If I don't find any other weird bugs with my logic I will submit a pull request.

I think I've pulled an earlier commit because I see FW 1.22 on the display, I have to check and clone again to have FW 1.23 locally and then apply my changes again.
